pkg/controller/restore_controller.go

```diff
@@ -85,6 +83,10 @@ type restoreController struct {
 	newBackupStore   func(*api.BackupStorageLocation, persistence.ObjectStoreGetter, logrus.FieldLogger) (persistence.BackupStore, error)
 }
 
+type restoreResult struct {
+	warnings, errors   api.RestoreResult
```

Comment:
There are extra spaces here, it'd be great if you could remove it :)

I think if you run ` go fmt ./... ` at the root of the project it should tell you about any misformating in the code. Or you could configure your editor to use `gofmt` on save.

Reply:
I remove it and rebased the commit